jupyterlab_hdf/baseHandler.py

```diff
@@ -76,28 +81,45 @@ def _handleErr(code, msg):
                 msg['type'] = 'JhdfError'
                 _handleErr(400, msg)
             except Exception as e:
-                msg = (f'Found and opened file, error getting contents from object specified by the uri.\n'
-                       f'Error: {traceback.format_exc()}')
+                msg = (
+                    f'Found and opened file, error getting contents from object specified by the uri.\n'
+                    f'Error: {traceback.format_exc()}'
+                )
                 _handleErr(500, msg)
 
             return out
 
+
 class HdfFileManager(HdfBaseManager):
-    """Implements base HDF5 file handling
-    """
+    """Implements base HDF5 file handling"""
+
     def _get(self, fpath, uri, **kwargs):
         with h5py.File(fpath, 'r') as f:
-            return self._getFromFile(f, uri, **kwargs)
+            hobj = self._getHobjFromFile(f, uri)
+            return self._getResponse(hobj, **kwargs)
 
-    def _getFromFile(self, f, uri, **kwargs):
+    def _getHobjFromFile(self, f, uri):
+        if uri == '/':
+            # Root is always resolvable
+            return f[uri]
+
+        link = f.get(uri, getlink=True)
+        if isinstance(link, h5py.ExternalLink):
+            return HobjExternalLink(name=uri, link=link)
+
+        return f[uri]
```

Comment:
Seems like I wasn't paying enough attention and I ran the formatting on this file :neutral_face: ... Sorry about that.

The main backend change is here: all previous calls to `f[uri]` now go through this method to check for external links. The `_getFromFile` method was renamed in `_getResponse` to avoid confusion with `_getHobjFromFile`.